llvm/test/Transforms/InstCombine/reduced.ll

```diff
@@ -0,0 +1,69 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+;; Tests that we do not always overwrite 
+; RUN: opt < %s -passes=instcombine -S | FileCheck %s
```

Comment:
Hmm, did you mean to add this test here...? It seems unrelated to this PR.

Reply:
I did not, my fault for `git add`ing a folder without checking my tree is clean...